### DISCUSSION TITLE
Readmoo

### DISCUSSION THREAD

**Discussion by brocaleb** [noDRM/DeDRM_tools#199] [Q&A]
How to use readmoo drm?

**Comment by JSWolf** [comment]
> How to use readmoo drm?

This is the best I could find.

https://github.com/dumo-dedrm/readmoo-dedrm

**Reply by DB99** [reply]
Server.js is a javascript file which you need to run,  Typically javascript files are run from a browser.  They can be run frem a command line, though often with some effort to get them working.  I'm no Java Programmer and won't even hazard a guess on this one.  If I was trying to run this personally I would start by downloading the repository, either with git clone or the zip file.  I would then go the relevant directlory and create an html file to run the js file.  Or I may try to run it from a command line using something like node in Linux.  A Google search for how to run javascript files should get you a releavnt link.  Don't expect it to work property first time.  If you do get it to work please report back to this thread showing what you did.  I have no expertise in Java, do not speak Mandarin or Cantonese and of course have no readmoo books and no readmoo app or program installed.  Figuring out the use of the program should be trivial to someone with a bit of javascript knowledge preopare to look at the code.  The Developer regarded how to run it as so trivial they did not give any instructions in the readme.

On another note, this is yet another DRM system in this case apparently linked to a large Taiwanese  book shop with apparently no English language support.  Given the available open source dedrm solution it should be relatively easy to add readmoo support in the plugin.  The question is whether it is worthwhile to do so.  There are now so many proprietary DRM solutions, especially for use in Android and/or IOS apps, that supporing removel of DRM in all of them even if possible would probably not be feasible or desirable.  How widespread is readmoo?

**Reply by brocaleb** [reply]
https://github.com/david-liu-950627/epub-exporter

A Chrome extension that exports EPUB file from Readmoo or books.com.tw.
